### ISSUE TITLE
kernel BUG: CPU stuck 

### ISSUE THREAD

**Issue by cselway** [ACAD-UofA/Bioinformatics-Wiki#15]
I just got this message on acad1 

[a1645778@acad1 170821_M03191_0227_000000000-B8K49]$ 
Message from syslogd@acad1 at Oct 18 16:02:19 ...
 kernel:BUG: soft lockup - CPU#14 stuck for 67s! [nfsiod:3830]

[a1645778@acad1 170821_M03191_0227_000000000-B8K49]$ 
Message from syslogd@acad1 at Oct 18 16:03:43 ...
 kernel:BUG: soft lockup - CPU#14 stuck for 67s! [quast.py:3650]

**Comment by grahamgower** [comment]
I don't think this reflects a real problem with the hardware. Its probably an issue in the OS caused by network connectivity issues associated with network maintenance. But I could be wrong. Please report any further observations.
